docs/source/userguide/configure/index.rst

```diff
@@ -352,6 +352,8 @@ There are some other parameters that you may need to specify:
 
 * TOTAL_JOBS: maximum number of jobs to be running at the same time in this platform.
 
+* LOG_RECOVERY_QUEUE_SIZE: A memory-consumption optimization for the recovery of logs. If not specified, defaults to min(100,TOTAL_JOBS) * 2, in case of issues with the recovery of logs, you can increase this value.
```

Comment:
I think the default is `200`.

`TOTAL_JOBS` doesn't have a default (above this line), so I would guess it to be `0` or `-1`... and then `min(100, TOTAL_JOBS`) would return `-1` instead of `100`?

(found the default, I think)

![image](https://github.com/user-attachments/assets/578887cc-f3e4-452f-ab4e-7ec9f4723675)

![image](https://github.com/user-attachments/assets/6a3a1a23-6769-4769-986f-baaa15800e25)

Reply:
That makes sense then :+1: